pkg/cloud/openstack/cluster/actuator.go

```diff
@@ -78,6 +79,10 @@ func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
 	if err != nil {
 		return err
 	}
+	loadbalancerService, err := loadbalancer.NewService(osProviderClient, clientOpts, clusterProviderSpec.UseOctavia)
```

Comment:
Can you create this services at the start of the method (where networkingService was created before)? It was intended to first create all clients, so that we don't start to do anything if one of them cannot be created (also in the other files)

Reply:
This method needs clusterProviderSpec.useOcatavia so that must after providerv1.ClusterSpecAndStatusFromProviderSpec.

Reply:
Oh of course you're right